Wifi credentials (#12793)
nrfconnect/sdk-nrf: doc/nrf/includes/wifi_credentials_static.txt

Wi-Fi static credential options
-------------------------------

> If you want to configure the credentials statically you can do so by setting the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_CREDENTIALS` Kconfig option to ``y``.

Reviewer: ```suggestion
If you want to configure the credentials statically, set the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_CREDENTIALS` Kconfig option to ``y``.
```

> .. important::
> 
>     it is not recommended to use static credentials in production environments.

Reviewer: ```suggestion
.. important::

   Do not use static credentials in production environments.
```

> Other options for statically configuring your wifi credentials:

Reviewer: ```suggestion
Other options for statically configuring your Wi-Fi credentials:
```
